.circleci/config.yml

```diff
@@ -93,15 +46,78 @@ anchor_7: &setup_bazel_remote_execution
       sudo bash -c "echo -e 'build --config=remote' >> .bazelrc.user";
       echo "Reading from remote cache for bazel remote jobs.";
 
+# Executor Definitions
```

Comment:
Might be a good idea to add links to docs on the first usage of these new features (executors, commands, parameters). At some point we're going to have to copy these into angular/angular and we might as well have the comments made already.